ROADMAP.md

```diff
@@ -19,33 +19,39 @@ The following is a list of prioritized roadmaps divided by OS (Linux/Windows) an
 
 **AMD Instinct**
 
-| Architecture | LLVM target | Sanity Tested | Release Ready |
-| ------------ | ----------- | ------------- | ------------- |
-| **CDNA3**    | **gfx942**  | ✅            |               |
-| CDNA2        | gfx90a      |               |               |
-| CDNA         | gfx908      |               |               |
-| GCN5.1       | gfx906      |               |               |
-| GCN5         | gfx900      |               |               |
+| Architecture | LLVM target | Build Passing | Sanity Tested | Release Ready |
+| ------------ | ----------- | ------------- | ------------- | ------------- |
+| CDNA3        | gfx942      | ✅            | ✅            | ✅            |
+| **CDNA2**    | **gfx90a**  |               |               |               |
```

Comment:
> Current focus areas are in **bold**. There will be exceptions from the "top-to-bottom" ordering occasionally based on test device availability.

I would add back bold for the latest architectures (CDNA3, RDNA4). The top of the list should typically always be bold, since we at least want support for the latest. I would also _remove_ bold from CDNA2, at least until we have the build passing and someone actively working on getting it tested and release ready.